validate.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	cid "github.com/ipfs/go-cid"
 	mh "github.com/multiformats/go-multihash"
+	_ "github.com/multiformats/go-multihash/register/blake3"
```

Comment:
We should remove this:
- It's already imported: https://github.com/multiformats/go-multihash/blob/89bf86c75b6b66226f63e22db775a72172970360/registry.go#L8
- https://github.com/multiformats/go-multihash/blob/89bf86c75b6b66226f63e22db775a72172970360/register/all/multihash_all.go#L21

Reply:
fixed